src/async/async_worker.c

```diff
@@ -281,7 +282,7 @@ __async_worker_op(WT_SESSION_IMPL *session, WT_ASYNC_OP_IMPL *op,
  * __async_worker --
  *	The async worker threads.
  */
-void *
+WT_THREAD_RET /* Quiet style.py */
```

Comment:
When I first hit an error in style.py, I thought it was a bug in my change so I made this hack. The real issue was the name of the function in the comment.

I will send a PR tomorrow to fix a bug in style.py related this issue (ie, pointers as return types confuse style.py)